### ISSUE TITLE
Add documentation for support-of/how-to-run back-annotated sims

### ISSUE THREAD

**Issue by ktbarrett** [cocotb/cocotb#1771]
According to at least two users (who both seem to be using Questa), cocotb can be used in back-annotated sims. We should advertise that point and have at least a basic how-to on it.

**Comment by themperek** [comment]
This in principle has not to do with cocotb. It is specific to the simulator and should be looking at the simulator user manual. can also us Verilog` $sdf_annotate` system task.

**Comment by ktbarrett** [comment]
Yeah it doesn't; I figured it would work without issue. But we should explicitly document it, otherwise potential end users might not realize it is supported. They don't really know *how* cocotb works, and how it isn't a problem.

**Comment by ktbarrett** [comment]
xref #522.
